boltdesignsystem/bolt
# DS-296 Image Element

## Jira

https://pegadigitalit.atlassian.net/browse/DS-296

## Summary

Adds an Image element that handles responsive images.

## Details

1. Image element is created with minimal CSS.
2. `fill` prop allows an image to always stretching to 100% width.
3. `background` prop transform the image into an absolute positioned background image. This removes the need to use the Background component.
4. Two CSS custom props are available to further adjust the background image fit and position.

## How to test

Run the branch locally and read through all the Image docs and demos under Elements. Make sure all props work as expected.
